groovy/groovy.debug/src/org/netbeans/modules/groovy/debug/GroovyBreakpointStratifier.java

```diff
@@ -78,7 +79,7 @@ public void stratify(JPDABreakpoint breakpoint) {
                 String printText = pt.replace("{groovyName}", (groovyName != null) ? groovyName : "?"); // NOI18N
                 String groovyPath = getGroovyPath(url, fo);
 
```

Comment:
Once Truffle out of the way, this stratifier gets called and adds Groovy stratum to the breakpoint.